lib/Word_Lib/Word_Lemmas.thy

```diff
@@ -895,26 +804,57 @@ lemma ucast_leq_mask:
   done
 
 lemma shiftl_inj:
```

Comment:
are these proofs faster or something? this is way longer than it used to be...

Reply:
This probably just needs a very different proof now because all the fundamentals changed, but I haven't done any of these, I don't really know.